clippy_lints/src/loops/for_loop_explicit_counter.rs

```diff
@@ -0,0 +1,56 @@
+use super::{get_span_of_entire_for_loop, make_iterator_snippet, IncrementVisitor, InitializeVisitor};
```

Comment:
`for_loop_expilicit_counter.rs` seems too redundant.
It would be preferable for the file name and lint name to match.